packages/node-cli/src/services/SuiteCloudAuthProxyService.js

```diff
@@ -100,6 +101,15 @@ class SuiteCloudAuthProxyService extends EventEmitter {
 			const localURL = `http://${LOCAL_HOSTNAME}:${proxyPort}`;
 			console.log(`SuiteCloud Proxy server listening on ${localURL}`);
 		});
+
+		this._localProxy.on('error', (error) => {
+			if (error.code === 'EADDRINUSE') {
+				const errorMsg = `Port ${proxyPort} is already being used. Choose a different port and try again.`;
+				console.error(errorMsg);
+				const emitObject = { message: errorMsg, authId: this._authId };
+				this.emit(EVENTS.ALREADY_USED_PORT, emitObject);
+			}
+		});
```

Comment:
I guess we missed to listen on proxy errors in the in initial implementation.
We might also want to emit other possible occurring errors, and not only the `'EADDRINUSE'`.
The error message we'll need to be moved to messages file.